app/processor/job.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package processor
+
+import (
+	"strings"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"github.com/mendersoftware/workflows/model"
+)
+
+type JobProcessor struct {
+	job *model.Job
+}
+
+type JsonOptions struct {
+}
```

Comment:
I don't understand the purpose of this.

Reply:
this is on purpose: to keep the interface consistent. in case there is a new round of refactoring (i.e.: a general processor interface) can we keep it?